### ISSUE TITLE
19.09 CopyQ doesn't start 

### ISSUE THREAD

**Issue by kuznero** [NixOS/nixpkgs#70896]
**Describe the bug**

```
ERROR: QtFatal: This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
ERROR:     (:0, )
terminate called after throwing an instance of '(anonymous namespace)::ExceptionQtFatal'
  what():  This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
 (:0, )
[1]    5804 abort (core dumped)  copyq
```

**To Reproduce**
1. use 19.09 channel
2. install copyq
3. try to start it

**Expected behavior**
The application starts without problems

**Metadata**

```
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.78, NixOS, 19.09.716.88bbb3c8096 (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3`
 - channels(root): `"nixos-19.09.716.88bbb3c8096"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: copyq
# a list of nixos modules affected by the problem
module:
```

**Comment by kuznero** [comment]
cc #65399

**Comment by c0bw3b** [comment]
Fixed in 222004e52e82ae7b827b20184d25c1ce88b85da6
